File: php/08_classes.php
<?php
// 08 classes

class Person {
    public $name;
    public $age;

    public function __construct($name, $age) {
        $this->name = $name;
        $this->age = $age;
    }

    public function greet() {
        return "hi i am " . $this->name . " and i am " . $this->age;
    }
}

$me = new Person("ulas", 20);
echo $me->greet() . "\n";
echo $me->name . "\n";

File: python/08_classes.py
# 08 simple class

class Person:
    def __init__(self, name, age):
        self.name = name
        self.age = age

    def greet(self):
        return f"hi i am {self.name} and i am {self.age}"

me = Person("ulas", 20)
print(me.greet())
print(me.name)
